internal/proxy/config.go

```diff
@@ -1,7 +1,20 @@
 package proxy
 
+import "time"
+
 type ServicesConfig struct {
-	Services []Config `yaml:"services" mapstructure:"services"`
+	EnvoyAgent EnvoyAgent `yaml:"envoy" mapstructure:"envoy"`
```

Comment:
let's update the config in https://github.com/goto/shield/blob/main/config/config.yaml with the example on how to use this envoy agent?